addons/mail/static/src/js/text_emojis_field.js

```diff
@@ -9,7 +9,10 @@ var MailEmojisMixin = require('mail.emoji_mixin');
 var _onEmojiClickMixin = MailEmojisMixin._onEmojiClick;
```

Comment:
This commit is more a [IMP] than a [ADD]

[ADD] is only for whole module additions.